connector/src/it/scala/com/datastax/spark/connector/rdd/RDDSpec.scala

```diff
@@ -12,7 +11,9 @@ import com.datastax.spark.connector.cql.CassandraConnector
 import com.datastax.spark.connector.embedded.SparkTemplate._
 import com.datastax.spark.connector.rdd.partitioner.EndpointPartition
 import com.datastax.spark.connector.writer.AsyncExecutor
+import spire.ClassTag
```

Comment:
Use scala.reflect.ClassTag

Reply:
Fixed in [[SPARKC-686] Fix incorrect import](https://github.com/datastax/spark-cassandra-connector/pull/1361/commits/cb36121b0f6c6142153b94dc09f19659ac26ffbe)